luarules/gadgets/mo_preventcombomb.lua

```diff
@@ -1,4 +1,4 @@
-local gadgetEnabled = not (Spring.GetModOptions().newdgun or Spring.GetModOptions().unba or Spring.GetModOptions().comupdate)
```

Comment:
You actually don't want this gadget enabled. This is responsible for making you lose when you d-gun another Commander.

Reply:
Isn't that why the "not" is in front?

Reply:
It's saying if any of the following modoptions are enabled, disable this gadget: new d-gun, unbacom, comm update. Removing that code means that the gadget will be enabled with the comm update modoption.

Reply:
Right, meant for it to be there, must have saved the wrong file, thanks for catching this!